hack/verify-argo-cd-versions.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+# To adopt a new version of Argo CD:
+# 1) Update this value to the GitHub tag of the target 'argoproj/argo-cd' release (example: 'v1.8.4'). 
+# 2) Fix the errors that are reported below (by editing the version string in the file reported in the error)
+TARGET_ARGO_CD_VERSION=v1.8.4
+
+# Extract the Argo CD repository string from ci-build.yaml, which SHOULD contain the target Argo CD version
+VERSION_FROM_CI_BUILD=$( awk '/repository: argoproj\/argo-cd/,/Pin a/' .github/workflows/ci-build.yaml )
```

Comment:
Does the awk script depends on the "# Pin a ..." command in ci-build.yaml?  If so, it would be safer to use some "marker" comment so that it does not get cleaned up or changed by other fellows.

Reply:
Done, thx @wtam2018